lib/light-service/organizer/with_reducer_log_decorator.rb

```diff
@@ -5,10 +5,10 @@ class WithReducerLogDecorator
 
       alias logged? logged
 
-      def initialize(organizer, decorated = WithReducer.new)
+      def initialize(organizer, decorated: WithReducer.new, logger:)
```

Comment:
If this class is used outside the gem, then this would be a breaking change. I can make it backwards compatible if you think it's worth it.

Reply:
It is an internal class, I think it's 👍.